about.html

```diff
@@ -0,0 +1,137 @@
+       <li><a href="index.html">About</a></li>
+       <li><a href="">program</a></li>
+       <li><a href="">News</a></li>
+       <li><a href="">Join</a></li>
+       <li><a href="">Sponsor</a></li>
+       <li><a href="">Compaign</a></li>
+     </ul>
+          <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        </div>
+    </nav>
+    <div class="wrapper">
+        
+      <div class="intro">
+        
+        <h3 class="title-about">"Hello Freshers!"</h3>
+        <h1 class="about-h1">Mobile first design strategy</h1>
+        <p class="about-info">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Impedit
+          magnam odit praesentium excepturi eveniet, dolor dolorem ex incidunt
+          perferendis
+        </p>
+        <p class="caption-about">2020/30/10 Join us for better knowledge</p>
+        <p class="caption-about1">@Kabul University Afghanistan</p>
+      </div>
+      <div class="main-program">
+        
+        <h2 class="main-program1">Main program</h2>
+        <div class="main">
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">Lecture</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur</p>
+        </div>
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">UI Exhibition</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit</p>
+        </div>
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">Forum</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur.</p>
+        </div>
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">Workshop</h3>
+          <p class="lecture-p">Lorem ipsum dolor</p>
+        </div>
+        <div class="lecturesection">
+          <i class="fa-duotone fa-podium"></i>
+          <h3 class="lecture">Ignit</h3>
+          <p class="lecture-p">Lorem ipsum dolor sit amet consectetur</p>
+        </div>
+        </div>
+        <a class="see" href="">See the whole program</a>
+       
+        <a class="button" href="">Get your Tickets</a>
+      </div>
+        <h2 class="feature-speaker">featured Speakers</h2>
+      <div class="featured">
+       
+
+        </div>   
+          
+        <div class="more">
+          <h4 class="more-speaker">more </h>
+          <img  class="more-img" src="images/Disabled.svg" alt="">
+        </div>
+
+      <div class="partner">
+        <h3 class="partners">Partner companies</h3>
+        <div class="companies">
+        <h6 class="google">Google</h6>
+        <h6 class="google">YouTube</h6>
+        <h6 class="google">Meta</h6>
+        </div>
+      </div>
+      <footer><p class="footer">@copyright UI-Ux designers</p></footer>
```

Comment:
- [ ] `required`
- [ ] Kindly align the speakers in the center as the design provided and a line below the speakers
- [ ] And please make the footer as similar as possible to the design, by adjust font size and spacing
- [ ] please see screenshots below

`your design`
- [ ] ![4page](https://user-images.githubusercontent.com/66230306/190279529-be31016d-8117-4f3d-a114-48bbd6a46a88.png)

`provided design`
![image](https://user-images.githubusercontent.com/66230306/190280076-527d0c40-3ec5-4e22-bd83-dff15b38b818.png)